src/routes.js

```diff
@@ -3,6 +3,24 @@
 var routes = {},
     util = require('util');
 
+var networkTexts = {
```

Comment:
I was wondering if we would even put them in the DB so I considered that as a temp. But it can be kept as a json somewhere.